test/integration/run_k2_cluster.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+set -e
+
+CPODIR=${CPODIR:=/tmp/___cpo_integ_test}
+rm -rf ${CPODIR}
+DEFAULT_EPS=("tcp+k2rpc://0.0.0.0:10000" "tcp+k2rpc://0.0.0.0:10001" "tcp+k2rpc://0.0.0.0:10002" "tcp+k2rpc://0.0.0.0:10003" "tcp+k2rpc://0.0.0.0:10004")
+EPS=( ${EPS:=${DEFAULT_EPS[@]}} )
+NUMCORES=${NUMCORES:=`nproc`}
+# core on which to run the TSO poller thread. Pick 4 if we have that many, or the highest-available otherwise
+#TSO_POLLER_CORE=$(( 5 > $NUMCORES ? $NUMCORES-1 : 4 ))
+# poll on random core
+TSO_POLLER_CORE=-1
+PERSISTENCE=${PERSISTENCE:=tcp+k2rpc://0.0.0.0:12001}
+CPO=${CPO:=tcp+k2rpc://0.0.0.0:9000}
+TSO=${TSO:=tcp+k2rpc://0.0.0.0:13000}
+COMMON_ARGS=${COMMON_ARGS:="--reactor-backend=epoll --enable_tx_checksum true --thread-affinity false"}
+HTTP=${HTTP:=tcp+k2rpc://0.0.0.0:20000}
+CPO_ADDITIONAL_PORT=${CPO_ADDITIONAL_PORT:=9001}
```

Comment:
This won't work - just remove this from here and the cmdline below. We don't need more than 1 CPO core anyway

Reply:
Done